source/extensions/filters/network/http_connection_manager/config.h

```diff
@@ -60,6 +60,22 @@ class HttpConnectionManagerConfigUtility {
                                            const Buffer::Instance& data);
 };
 
+/**
+ * Determines if an address is internal based on user provided config.
+ */
+class InternalAddressConfig : public Http::InternalAddressConfig {
+public:
+  InternalAddressConfig(const envoy::config::filter::network::http_connection_manager::v2::
+                            HttpConnectionManager::InternalAddressConfig& config);
+
+  bool isInternalAddress(const Network::Address::Instance& address) const override {
+    return address.type() == Network::Address::Type::Pipe && unix_sockets_;
```

Comment:
This gets tricky. Because you haven't yet added a setting for which network subnets are internal vs external, I think you need to call the existing code here (at least if the address.type() == Ip4/Ip6). Otherwise, if you set this to true, ONLY unix domain sockets are considered internal.

Reply:
Yeah that sounds like a good idea until the ip subnet parts can be added to the config